modules/display-books.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable import/prefer-default-export */
+const container = document.querySelector('.collection');
+const bookList = JSON.parse(localStorage.getItem('book-list')) || [];
+export const displayBlock = () => {
+  container.innerHTML = '';
+  bookList.forEach((book, index) => {
+    container.innerHTML += `<ul class="Books">
+      <li>${book.title}</li>
+      <li>${book.author}</li>
```

Comment:
[OPTIONAL] I think it would be better if you separate the book and the author to increase the UX of your app. Instead of "BookAuhor",  "Book by Author" would be better.
<img width="791" alt="Screenshot 2023-03-27 at 7 49 23 PM" src="https://user-images.githubusercontent.com/25887253/228016105-dabb3a76-fbfb-40b7-9953-0abacd411c37.png">